data/categories.txt

```diff
@@ -1614,6 +1614,12 @@ el:Είδη κιγκαλερίας|DIY|Κάν'το μόνος σου|κατάσ
 sk:Železiarstvo|Náradie
 fa:فروشگاه ابزار و یراق
 
+shop-houseware|@shop
+en:4Houseware
+be:4Пасуда|4Бытавыя тавары
+ru:4Посуда|4Бытовые товары
+uk:4Посуд|4Побутові товари
+
```

Comment:
```suggestion
nl: 4Huishouden
```

Reply:
```
    en = Housewares
    ar = بضائع منزلية
    be = Бытавыя тавары
    bg = Домакински стоки
    ca = Articles de la llar
    cs = Domácí potřeby
    da = Husholdningsartikler
    de = Haushaltswaren
    el = Οικιακά αγαθά
    es = Enseres domésticos
    es-MX = Artículos de uso doméstico
    et = Kodutarbed
    eu = Etxeko ondasunak
    fa = کالاهای خانگی
    fi = Kotitaloustavarat
    fr = Articles ménagers
    he = מוצרים לבית
    hi = घरेलू सामान
    hu = Háztartási cikkek
    id = Barang-barang rumah tangga
    it = Articoli per la casa
    ja = 家庭用品
    ko = 가정용품
    mr = घरगुती वस्तू
    nb = Husholdningsartikler
    nl = Huishoudelijke artikelen
    pl = Artykuły gospodarstwa domowego
    pt = Bens de uso doméstico
    pt-BR = Bens domésticos
    ro = Bunuri de uz casnic
    ru = Бытовые товары
    sk = Tovar pre domácnosť
    sv = Hushållsartiklar
    sw = Bidhaa za nyumbani
    th = สินค้าใช้ในบ้าน
    tr = Ev eşyaları
    uk = Побутові товари
    vi = Hàng gia dụng
    zh-Hans = 家庭用品
    zh-Hant = 家庭用品
```

Reply:
es = Enseres domésticos
es-MX = Enseres domésticos

Reply:
So you want to replace type_strings or categories or both?

Reply:
Do not replace, fill in just missing parts.